examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    user_id_raw = ["user_id"] &gt;&gt; Rename(postfix='_raw') &gt;&gt; LambdaOp(lambda col: col.astype("int32")) &gt;&gt; TagAsUserFeatures()
</pre><p>I think we no longer need <code>user_id_raw</code> and <code>item_id_raw</code> since we do not register them in the feature store.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>